modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ConfigurationContainer.tsx

```diff
@@ -79,6 +79,22 @@ export function ConfigurationContainer({
 				}
 				toggled={values.enableObjectEntryHistory}
 			/>
+
+			{Liferay.FeatureFlags['LPS-181663'] && (
+				<Toggle
+					disabled={isReadOnly}
```

Comment:
Hey @devpedrotavares , do you know how to inform the expected behavior for system objects? Also, does the user need any permissions to be able to change the value of this toggle?

Reply:
If I create a new user with only view permissions, that user can try to update this field. But the save and publish buttons are disabled for him. So I believe we need to add the update permission in this disabled property.

![screenshot-1693832020](https://github.com/liferay-objects/liferay-portal/assets/61854510/560191e0-161c-49ff-b47a-b01a1c8238f7)

@nara Enable Entry History in Audit Framework has the same behavior, could you confirm if this is a bug? If so, do we already have a bug open to change the field Enable Input History in Audit Framework Without Permission?

Reply:
@igor-franca  Yeah, this seems a bug to me because it doesn't make sense to have those toggles enabled without being possible to save or publish the definition. Regarding if we already have a bug opened about it, I suggest you open a thread in the BPM Slack channel and tag the QA squad A to confirm it.

Reply:
Hey @igor-franca. This feature is just for custom objects. 
And I agree with you, it makes sense to pass the update permission in the disable property. I just added it.